requirements/app.txt

```diff
@@ -1,9 +1,9 @@
-galaxy.plugin.api==0.69
+galaxy.plugin.api
 
 # steam_network backend
-protobuf==3.19.4
```

Comment:
any version of protobuf after this drops support for python 3.7, so we need this

Reply:
ah, fair enough. I was attempting to actually use my Python 3.8 and get everything working under that, but might be too big a conversion job...

Reply:
I'd prefer it under python 3.11 if i'm being honest. 3.10 added match, and coming from OOP languages not having a switch statement is brutal. But GOG Galaxy uses 3.7 internally, so that's what we need to target

Reply:
reverted